posts/Gunnar/2018-03-19-spec-api-modularity-patterns.adoc

```diff
@@ -0,0 +1,426 @@
+= "Spec API Modularity Patterns"
```

Comment:
Why quotes on the title?

Reply:
Weird, all my posts since last year have this. I one did this for some reason I don't remember, and always kept it when copying this over to the next post. It's not rendered, though. Thanks for pointing out :)